### PR TITLE
Adding note about accessing the VPN behind country level firewalls

### LINKED ISSUE #1415: Explain the potential network connections issues of devices from China
As a fleet owner with devices in China, I'd like to have more information on which features of balenaCloud would have issues due to the great firewall of China.

Adding a bit more detail here might be the best place https://www.balena.io/docs/reference/OS/network/2.x/#network-requirements

We heard that VPN connection from China kinda working but we also observed that some users have issues because of China's firewall rules. This is a known problem and there are several users reporting this: https://github.com/balena-io/balena/issues/466

### PR DESCRIPTION
**Pull request by garethtdavies**
Closes #1415 

Added some general text about accessing the VPN behind country-level firewalls and also a specific FAQ with reference to China. What else should be in here cc @alisondavis17 

Change-type: patch
Signed-off-by: Gareth Davies <gareth@balena.io>

### CODE REVIEW

File: pages/faq/troubleshooting/faq.md [hunk 1]
```diff
@@ -78,6 +77,10 @@ Additionally, you should whitelist the following domains for the relevant ports
 * `*.docker.com`
 * `*.docker.io`
 
+##### Can I use {{ $names.cloud.lower }} in China?
```
Comment (xginn8) [inline pages/faq/troubleshooting/faq.md:80]:
```suggestion
##### Can I use {{ $names.cloud.lower }} in countries with restrictive national firewalls?
```

Comment (xginn8) [reply pages/faq/troubleshooting/faq.md:80]: might be better to avoid naming names, just a thought!

Comment (garethtdavies) [reply pages/faq/troubleshooting/faq.md:80]: That was my thought too, which is why I only included that bit in a FAQ section, where it probably is appropriate but I'll let someone else decide :)

Comment (alisondavis17) [reply pages/faq/troubleshooting/faq.md:80]: I suspect some people will search for China specifically; that's the country we tend to get asked about. Maybe "countries with restrictive firewalls such as China?"

**Comment by garethtdavies** [comment]
@balena-ci retest

**Comment by garethtdavies** [comment]
@balena-ci rebase

**Pull request merged**